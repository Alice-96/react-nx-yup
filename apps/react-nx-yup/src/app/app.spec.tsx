import { render } from '@testing-library/react';

import App from './app';

describe('App', () => {
  it('render', () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
