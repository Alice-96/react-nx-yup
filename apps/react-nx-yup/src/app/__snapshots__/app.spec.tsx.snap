// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
    >
      <button
        aria-label="menu"
        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart MuiIconButton-sizeMedium css-134qg7o-MuiButtonBase-root-MuiIconButton-root"
        tabindex="0"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="MenuIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
      <div
        class="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
      >
        YupTrial
      </div>
    </div>
  </header>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-dver4o-MuiContainer-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-t5ouwv-MuiPaper-root"
    >
      <h2
        class="MuiTypography-root MuiTypography-h6 css-oblp1g-MuiTypography-root"
      >
        Links
      </h2>
      <div>
        <div>
          <a
            href="/yup"
          >
            yup
          </a>
        </div>
        <div>
          <a
            href="/yupNested"
          >
            yupNested
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
